pipelines/live-1/main/check-environment.yaml

```diff
@@ -1,28 +1,18 @@
 resources:
-- name: cloud-platform-environments-repo
-  type: git
-  source:
-    uri: https://github.com/ministryofjustice/cloud-platform-environments.git
-    branch: master
-    git_crypt_key: ((cloud-platform-environments-git-crypt.key))
-- name: tools-image
+- name: ruby-image
   type: docker-image
   source:
-    repository: 926803513772.dkr.ecr.eu-west-1.amazonaws.com/cloud-platform/tools
-    tag: latest
-    aws_access_key_id: ((aws-live-0.access-key-id))
-    aws_secret_access_key: ((aws-live-0.secret-access-key))
+    repository: registry.hub.docker.com/ministryofjustice/orphaned-namespace-checker
+    tag: 1.21
 - name: slack-alert
   type: slack-notification
   source:
     url: https://hooks.slack.com/services/((slack-hook-id))
 - name: every-week
   type: time
   source:
-    interval: 10m
-    days: [Wednesday]
-    start: 16:40
-    stop: 16:50
+    # number of minutes in a week
+    interval: 10080m
```

Comment:
The old way of specifying this felt easier to reason with, is there a reason to convert it all to minutes? If so, perhaps in hours it would be a more familiar number?

Reply:
Agreed, but it had stopped working. We moved this to hours as suggested.